UPGRADING.INTERNALS

```diff
@@ -37,6 +37,9 @@ PHP 8.4 INTERNALS UPGRADE NOTES
 * The zend_*printf family of functions now supports the "%S" modifier to print
   out zend_string*. This won't cut off the string if it embeds a NUL byte.
 
+* zend_mm_set_custom_debug_handlers() has been removed from ZendMM, use
+  zend_mm_set_custom_handlers() instead
+
```

Comment:
I think we should probably talk about the `ZEND_API` functions like `__zend_malloc` which have different signatures in debug builds now. These won't matter for most developers, but there are a few (like those using Rust) which may want to know about such changes.